controllers/componentdetectionquery_controller.go

```diff
@@ -111,31 +111,36 @@ func (r *ComponentDetectionQueryReconciler) Reconcile(ctx context.Context, req c
 		source := componentDetectionQuery.Spec.GitSource
 		var devfileBytes, dockerfileBytes []byte
 		var clonePath string
+		var componentPath string
 		devfilesMap := make(map[string][]byte)
 		devfilesURLMap := make(map[string]string)
 		dockerfileContextMap := make(map[string]string)
 
+		context := source.Context
+		contextDir := context
+		if context == "" {
+			contextDir = "./"
+		}
```

Comment:
```suggestion
		context := source.Context
		if context == "" {
			context = "./"
		}
```

why not this and replace all `contextDir` occurances

Reply:
updated.